tests/unit/plugins/test_base.py

```diff
@@ -17,10 +16,6 @@
 from tuned import storage
 import tuned.plugins.base
 
-tuned.plugins.base.log = flexmock.flexmock(info = lambda *args: None,\
-	error = lambda *args: None,debug = lambda *args: None,\
-	warn = lambda *args: None)
-
```

Comment:
Yes, this is exactly what I was hoping keeping the mocking in globals.py would allow us to do. Thanks for looking into it!